cnxpublishing/views/admin.py

```diff
@@ -271,3 +295,250 @@ def admin_edit_site_message_POST(request):
     args = admin_edit_site_message(request)
     args['response'] = "Message successfully Updated"
     return args
+
+
+def get_baking_statuses_sql(request):
```

Comment:
Can you put in a docstring saying what this does and what it returns. It doesn't need to be involved, just simple and straightforward.